examples/input/touch_input.rs

```diff
@@ -0,0 +1,40 @@
+use bevy::{input::touch::*, prelude::*};
+
+fn main() {
+    App::build()
+        .add_default_plugins()
+        .init_resource::<TouchHandlerState>()
+        .add_system(touch_system.system())
+        .run();
+}
+
+#[derive(Default)]
+struct TouchHandlerState {
+    touch_event_reader: EventReader<TouchInput>,
+}
+
+fn touch_system(
```

Comment:
Once we make "high level" touch changes, I think we should have two touch examples. One that illustrates how to use touch events (like the current example does) and one that illustrates the high level interface.